File: src/main/java/econtroller/actuator/ActuatorInit.java
package econtroller.actuator;

import econtroller.ControllerConfiguration;
import se.sics.kompics.Init;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-04-23
 *
 */

public class ActuatorInit extends Init {

	private ControllerConfiguration controllerConfiguration;

	public ActuatorInit(ControllerConfiguration controllerConfiguration) {
		this.controllerConfiguration = controllerConfiguration;
	}

	public ControllerConfiguration getControllerConfiguration() {
		return controllerConfiguration;
	}

}

File: src/main/java/econtroller/actuator/ActuatorInit.scala
package econtroller.actuator

import se.sics.kompics.Init
import econtroller.ControllerConfiguration

/**
 * @author Amir Moulavi
 * @date 2011-07-23
 *
 */
class ActuatorInit(val controllerConfiguration:ControllerConfiguration) extends Init {
  def getControllerConfiguration = controllerConfiguration
}